Sources/SnowManLogic/ViewLogic/MockView.swift

```diff
@@ -0,0 +1,40 @@
+public class MockView: Viewable {
+	var ui: Receivable 
+	var guess = ""
```

Comment:
We discussed a preferred coding style for handling optional variables, and that empty variables are a code smell
```swift
public var foo: Type?
```
or
```swift
public var foo: Type!
```